pkg/skaffold/event/event.go

```diff
@@ -712,3 +712,17 @@ func BuildSequenceFailed(err error) {
 	handler.state.BuildState.StatusCode = aiErr.ErrCode
 	handler.stateLock.Unlock()
 }
+
+func SkaffoldInitFailed(err error) {
+	handler.logEvent(proto.LogEntry{
```

Comment:
`logEvent` doesn't seem to be piped through the `eventChan` of the `eventHandler`. Wonder if that can cause this specific event to occur out of order?

Reply:
could be. Let me look in to it.